vtex-apps/minicart
# Feature popup hover btn with link

#### What problem is this solving?

The client has the minicart set as "popup" and "hover", but there was a need for it to act as a link for the checkout when clicked.

#### How to test it?

Add a product to the cart, check the minicart, when clicking on the minicart icon the user must be redirected to checkout

[Workspace](https://minicarttestes--tokstokio.myvtex.com/)

#### Screenshots or example usage:

[Video](https://drive.google.com/file/d/1iaikLe0lH4QzkVvhxMIjq9ZNk0nJtotV/preview
)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/u4EQe6mJf0nTXe3dOx/giphy.gif)
